From 9758de9db578ac24b7000b7aa08fea243eb72227 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:13:04 -0400
Subject: [powerpc] powernv: Add missing include to LPC code

Message-id: <1410545655-205645-556-git-send-email-dzickus@redhat.com>
Patchwork-id: 94604
O-Subject: [RHEL7 PATCH 555/626] powerpc/powernv: Add missing include to LPC code
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 0c0a3e5a100bbc4aaedd140e82b429227a76701b
Author: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Date:   Sat Jun 7 07:02:14 2014 +1000

    powerpc/powernv: Add missing include to LPC code

    kbuild bot spotted that one:

      arch/powerpc/platforms/powernv/opal-lpc.c: In function 'opal_lpc_init_debugfs':
    >> arch/powerpc/platforms/powernv/opal-lpc.c:319:35: error: 'powerpc_debugfs_root' undeclared (first use in this function)
         root = debugfs_create_dir("lpc", powerpc_debugfs_root);
                                          ^
    We neet to include the definition explicitely.

    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/platforms/powernv/opal-lpc.c b/arch/powerpc/platforms/powernv/opal-lpc.c
index 70eff22..f04b4d8 100644
--- a/arch/powerpc/platforms/powernv/opal-lpc.c
+++ b/arch/powerpc/platforms/powernv/opal-lpc.c
@@ -22,6 +22,7 @@
 #include <asm/opal.h>
 #include <asm/prom.h>
 #include <asm/uaccess.h>
+#include <asm/debug.h>
 
 static int opal_lpc_chip_id = -1;
 
-- 
1.7.1